src/components/Registration/Registration.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import Button from '../../common/Button/Button';
+import Input from '../../common/Input/Input';
+import './styles.scss';
+
+export default function Registration() {
+	const [name, setName] = useState('');
+	const [email, setEmail] = useState('');
+	const [password, setPassword] = useState('');
+	const navigate = useNavigate();
+
+	const registerUser = async (e: any) => {
+		e.preventDefault();
+		const newUser = {
```

Comment:
It make sense double check, that data for the backend are ok, so, could you add some UI verification of fields?